ykrt/pass_finder/try_passes.py

```diff
@@ -369,6 +376,6 @@ def main(logf, glogf, is_prelink):
     CWD = args.path
     print(f"PATH to interpreter: {CWD}")
 
-    with open("genetic.log", "w+") as glogf:
-        with open("passes.log", "w") as logf:
+    with open("/home/shreei/research/yk_pv/ykrt/pass_finder/genetic.log", "w+") as glogf:
```

Comment:
Not convinced this line will work on my machine either ;)

Reply:
This too.